core_data_modules/logging/logging.py

```diff
@@ -15,11 +15,18 @@ class LogLevels(object):
 
 
 class Logger(object):
+    project_name = ""
+    
     def __init__(self, name):
         self.name = name
 
+    @classmethod
+    def set_project_name(cls, project_name):
+        cls.project_name = project_name
+
     def log(self, log_level, message):
-        print(f"{TimeUtils.utc_now_as_iso_string()} - {log_level.string_value} - {self.name}: {message}")
+        print(f"{TimeUtils.utc_now_as_iso_string()} {log_level.string_value} - "
+              f"{self.project_name}{'/' if self.project_name is not '' else ''}{self.name}: {message}")
```

Comment:
I would generally move the 3 way conditional onto a separate line. This feels a bit too much inside a string.

Reply:
agreed; moved